Library/Formula/ice.rb

```diff
@@ -18,6 +30,34 @@ class Ice < Formula
   patch :DATA
 
   def install
+    resource("berkeley-db").stage do
+
+      (Pathname.pwd).install resource("berkeley-db-patch")
+      safe_system "/usr/bin/patch", "-g", "0", "-f", "-d", Pathname.pwd, "-p1", "-i", "berkeley-db-5.3.28.diff"
+
+      # BerkeleyDB dislikes parallel builds
+      ENV.deparallelize
+      # --enable-compat185 is necessary because our build shadows
+      # the system berkeley db 1.x
+      args = %W[
+        --disable-debug
+        --prefix=#{prefix}
+        --mandir=#{man}
```

Comment:
@cscetbon Yeah, I think I went away to make a drink and came back to find you'd fixed most of what I'd just commented :smile_cat:.

Looks good. The replacement of the patch with the inreplace was a good step, much less overhead to deal with that way, nice one. Just a few more bits and I think we'll finally be done.